src/api/service/axios-bitbucket-client.ts

```diff
@@ -44,7 +44,8 @@ export class AxiosBitBucketClient {
   }
 
   public createBranch(projectKey: string, repositorySlug: string, name: string, startPoint: string) {
-    return this.client.post(`rest/api/1.0/projects/${projectKey}/repos/${repositorySlug}/branches`, {name, startPoint, message: 'Create release branch'}).then(response => {
+    return this.client.post(`rest/api/1.0/projects/${projectKey}/repos/${repositorySlug}/branches`, {name, startPoint, message: 'Create release branch'})
+    .then(response => {
```

Comment:
How did you identified those formatting/linting issues? do we need to update the linting rules?

Reply:
Throughout the repo, it was inconsistent on whether .then and .catch should be on a new line or at the end of the previous line, sometimes within the same function. I moved them all to new lines, as I feel this improves the readability; do thing X, and then afterwards do thing Y.
I'm not sure how the linter could incorporate this, but I could dig into it - maybe saying that .+\.then and .+\.catch are no good?

Reply:
Okay, having looked into it adding a new eslint module is massively complicated - far more effort than would be valuable for a small layout convention.